data/mods/Tamable_Wildlife/insect_spider.json

```diff
@@ -0,0 +1,380 @@
+[
+  {
+    "id": "mon_plague_nymph",
+    "type": "MONSTER",
+    "copy-from": "mon_plague_nymph",
+    "name": { "str": "plague nymph" },
+    "petfood": {
+      "food": [ "BUGFOOD" ],
+      "feed": "The %s seems to... uhh, like you?  Well, as certainly as you could tell a giant bug could like you.  It skitters around your legs and seems friendly.",
+      "pet": "The %s skitters around your feet."
+    },
+    "extend": { "flags": [ "CANPLAY" ] }
+  },
+  {
+    "id": "mon_skittering_plague",
+    "type": "MONSTER",
+    "copy-from": "mon_skittering_plague",
+    "name": { "str": "skittering plague" },
+    "extend": { "flags": [ "CANPLAY" ] }
+  },
+  {
+    "id": "mon_plague_vector",
+    "type": "MONSTER",
+    "copy-from": "mon_plague_vector",
+    "name": { "str": "plague vector" },
+    "extend": { "flags": [ "CANPLAY" ] }
+  },
+  {
+    "id": "mon_giant_cockroach_nymph",
+    "type": "MONSTER",
+    "copy-from": "mon_giant_cockroach_nymph",
+    "name": { "str": "giant cockroach nymph" },
+    "petfood": {
+      "food": [ "BUGFOOD" ],
+      "feed": "The %s seems to... uhh, like you?  Well, as certainly as you could tell a giant bug could like you.  It skitters around your legs and seems friendly.",
+      "pet": "The %s skitters around your feet."
+    },
+    "extend": { "flags": [ "CANPLAY" ] }
+  },
+  {
+    "id": "mon_giant_cockroach",
+    "type": "MONSTER",
+    "copy-from": "mon_giant_cockroach",
+    "name": { "str": "giant cockroach", "str_pl": "giant cockroaches" },
+    "extend": { "flags": [ "CANPLAY" ] }
+  },
+  {
+    "id": "mon_pregnant_giant_cockroach",
```

Comment:
Will its nymphs also be friendly when it dies 🤔

Reply:
Cockroach nymphs aren't aggressive, they'd just keep running away. There's probably some way to have it spawn already tamed nymphs when dead but I am not 100% sure that's json territory? Though to be fair I haven't looked at how cockroaches handle this in the first place, this could be interesting.

Reply:
Okay, so I've asked around and looked through some code. There _probably_ is a way to make them spawn already tamed nymphs but that'd require some magiclysm fuckery and I am not entirely sure that's even manageable without some additional c++ (in magiclysm the spells are cast by the player, here they'd be cast by a dying cockroach, I am not sure if there's a difference but I assume there is)